cluster-state-service/internal/features/wrappers/ec2_wrapper.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 const (
-	ec2InstanceLaunchRetrySleep = 1
+	ec2InstanceLaunchRetrySleep = 2
```

Comment:
Is this in seconds? Making it clear in the name will be nice. And using Time.Second etc. is recommended.